classes/decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super()
```

Comment:
- [ ] You do not need to call `super()`. You can user `attr_accessor :nameable` instead.

Reply:
Done.